Assets/Scripts/Environment/BreakableWall.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BreakableWall : MonoBehaviour
+{
+    /* Notes:
+     * The wall object with this script must also have a collider component
+     */
+    public int wallHealth = 10;
+    public int playerCollisionDamage= 5;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (wallHealth <= 0)
```

Comment:
As stated before this check should be done in the TakeDamage function so the update method should be removed since it is not needed.